src/libraries/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/HKDF.cs

```diff
@@ -232,10 +247,15 @@ public static byte[] DeriveKey(HashAlgorithmName hashAlgorithmName, byte[] ikm,
         /// <param name="output">The output buffer representing output keying material.</param>
         /// <param name="salt">The salt value (a non-secret random value).</param>
         /// <param name="info">The context and application specific information (can be an empty span).</param>
+        /// <exception cref="ArgumentException"><paramref name="ikm"/> length is larger than allowed length.</exception>
+        /// <exception cref="ArgumentOutOfRangeException"><paramref name="output"/> has a length less than 1.</exception>
```

Comment:
```suggestion
        /// <exception cref="ArgumentException"><paramref name="output"/> is empty.</exception>
```

Updated to match the other place that's throwing because of an empty span.